talk/expert.tex

```diff
@@ -403,11 +403,45 @@
       struct hasFoo<T, decltype(std::declval<T>().foo())>
         : std::true_type {};
 
-      std::cout << hasFoo<MyType>::value << std::endl;
+      struct A{}; struct B{void foo();}; struct C{int foo();};
+      static_assert(!hasFoo<A>::value, "A has no Foo()");
+      static_assert(hasFoo<B>::value, "B has Foo()");
+      // static_assert(hasFoo<C>::value, "C has Foo()"); fails assertion!
     \end{cppcode*}
   \end{exampleblock}  
 \end{frame}
 
+\begin{frame}[fragile]
+  \frametitleii{Using \texttt{void\_t} to simplify SFINAE expressions \hfill \cpp17}
```

Comment:
frametitleii will put C++11 in the title but you add also C++ 17 manually. This is contradictory.
I think you want to use frametitleia (sorry for the weird leet coding but latex does not allow numbers in commands and frametitleseventeen was too long for me...)